config/config.go

```diff
@@ -134,6 +141,10 @@ var defaultConf = Config{
 		Capacity: 2560,
 		Shards:   256,
 	},
+	PreparedPlanCache: PreparedPlanCache{
+		Enabled:  true,
```

Comment:
how about disable plan cache for prepare statements by default ?

Reply:
If you have any concern about enabling this feature by default, I'll change its default value.